lightningd/options.c

```diff
@@ -672,6 +672,16 @@ static void check_config(struct lightningd *ld)
 
 	if (ld->use_proxy_always && !ld->proxyaddr)
 		fatal("--always-use-proxy needs --proxy");
+
+	if (ld->proxyaddr && (!ld->use_proxy_always || ld->config.use_dns)) {
+		if (tal_count(ld->proposed_wireaddr) != 0 && some_tor_addresses(ld->proposed_wireaddr)) {
+			log_unusual(ld->log, "WARNING: Your Tor proxy setup with use of hidden service address"
+						" and --allways-use-proxy=%s and --disable-dns=%s"
```

Comment:
spelling: --always-use-proxy, --disable-dns (without argument)

Reply:
WARNING: Your Tor setup use a hidden service address --always-use-proxy=**false** and --disable-dns **not set** is not recommended, please set --always-use-proxy=true and use --disable-dns, we hope you know what you are doing!
 c-lightning try to help you and detected that you have configured a possible dns leak of your hidden service

Reply:
Thanks, much more understandable.

WARNING: Your Tor setup use**s** a hidden service address **with** --always-use-proxy=false and **without** --disable-dns **which** is not recommended, please set --always-use-proxy=true and use --disable-dns, we hope you know what you are doing!
c-lightning **is trying** to help you **because it** detected that you have configured a possible dns leak of your hidden service

Reply:
Thank you too, you caught one, but now its enough for me, since I fear that ends in tautologies.
Some say I am as polite and hansom as a satoshi, but I clearly differ when there is no real progress, then I tent to get rude and bossy.

Here is the bossy unfair mean and I feel a shame for exscript of the original draft of this PR and some hints for the next party, where some caught the front row  tick tick ...

> Cybermen have tagged your system for upgrade by low r value and the doctor is out of town ...

Reply:
There was no ill intent in my posts. We can leave it as it is.

Reply:
keep CALM,  You are clearly welcome and even now there is one mistype, if you insist, I will change the p to P to make you happy

Reply:
i dont insist on anything - its your call. And this PR does not affect my feelings whatsoever.